actions/add-review-url/package.json

```diff
@@ -1,29 +1,29 @@
 {
-  "name": "AddReviewUrl",
+  "name": "addcomment",

```

Comment:
Why change this?

Reply:
Updated. (Had created and tested a new Typescript package.json)